### PR TITLE
Casassarnau/baggage api

### PR DESCRIPTION
**Pull request by Casassarnau**
## What this PR does / Why we need it
A few sentences describing the overall goals of the pull request's commits.

## Which issue(s) this PR fixes (optional) 

Fixes #

## Some questions
- [x] I have read the contributing guidelines
- [x] I abide by this repository Code of Conduct
- [x] I understand that my PR won't be merged until Travis gives a "green light"

## Additional Notes (optional)
Do you want to add anything else? We :heart: to hear your opinions!

### CODE REVIEW

**Comment by casassg** [comment]
./baggage/views.py:25:1: E302 expected 2 blank lines, found 1
./baggage/views.py:61:21: W291 trailing whitespace
./baggage/views.py:71:32: E127 continuation line over-indented for visual indent
./baggage/views.py:74:17: W291 trailing whitespace
./baggage/views.py:88:25: E127 continuation line over-indented for visual indent
./baggage/views.py:255:1: E302 expected 2 blank lines, found 1
./baggage/views.py:257:34: E251 unexpected spaces around keyword / parameter equals
./baggage/views.py:257:36: E251 unexpected spaces around keyword / parameter equals
./baggage/views.py:266:121: E501 line too long (160 > 120 characters)
./baggage/views.py:273:121: E501 line too long (200 > 120 characters)
./baggage/views.py:273:173: E231 missing whitespace after ':'
./baggage/views.py:273:190: E231 missing whitespace after ':'
./baggage/views.py:275:1: W293 blank line contains whitespace
./baggage/views.py:291:1: W293 blank line contains whitespace
./baggage/views.py:292:1: W293 blank line contains whitespace
./baggage/views.py:293:1: W293 blank line contains whitespace
./baggage/views.py:294:1: W293 blank line contains whitespace
./baggage/views.py:295:1: W391 blank line at end of file
./baggage/views.py:295:1: W293 blank line contains whitespace
./baggage/views.py:295:1: E303 too many blank lines (4)
./judging/views.py:110:121: E501 line too long (126 > 120 characters)
./checkin/views.py:23:1: E302 expected 2 blank lines, found 1
./checkin/views.py:40:25: E127 continuation line over-indented for visual indent
./checkin/views.py:110:14: E111 indentation is not a multiple of four
./checkin/views.py:111:40: E127 continuation line over-indented for visual indent
./checkin/views.py:117:1: E302 expected 2 blank lines, found 1
./checkin/views.py:131:34: E251 unexpected spaces around keyword / parameter equals
./checkin/views.py:131:36: E251 unexpected spaces around keyword / parameter equals
./checkin/views.py:134:44: W291 trailing whitespace
./checkin/views.py:140:1: W293 blank line contains whitespace
./checkin/views.py:143:24: E222 multiple spaces after operator
./checkin/views.py:148:1: W293 blank line contains whitespace
./checkin/views.py:149:1: W391 blank line at end of file
./checkin/views.py:149:1: W293 blank line contains whitespace
./applications/models.py:73:1: E303 too many blank lines (3)
./app/emails.py:3:1: F401 'django.template.Context' imported but unused

File: checkin/views.py [hunk 1]
```diff
@@ -1,9 +1,15 @@
+#!/usr/local/bin/python
```
Comment (casassg) [inline checkin/views.py]: Remove this pls

File: checkin/views.py [hunk 2]
```diff
@@ -13,10 +19,54 @@
...
+                                       'before going to the infodesk :information_source: at '
+                                       'my.hackupc.com.\n    • If you need technical :three_button_mouse: '
+                                       'help, ask a mentor :female-teacher::skin-tone-3: at '
+                                       'mentors.hackupc.com.\nRemember that if I\'m unable to answer '
+                                       ':speak_no_evil:, you can try with the <#' +
+                                       settings.SLACK_BOT.get('channel', None) + '> channel '
+                                       ':speech_balloon: or to any organizer :tshirt: around.\n'
+                                       '*If there\'s any emergency :rotating_light: seek for any organizer, '
+                                       'you can also ping <@' + settings.SLACK_BOT.get('director1', None) +
+                                       '> or <@' + settings.SLACK_BOT.get('director2', None) + '>.*')
```
Comment (casassg) [inline checkin/views.py]: - Use multi-line strings
- Replace dots with `-`

File: checkin/views.py (hunk 1 above)
Comment (Casassarnau) [reply checkin/views.py]: voy

File: baggage/views.py [hunk 3]
```diff
@@ -18,6 +19,74 @@
...
+
+
+def baggage_checkOut(request, web):
+    bagid = request.POST.get('bag_id')
+    bag = Bag.objects.filter(bid=bagid).first()
+    bag.status = BAG_REMOVED
+    if web:
+        bag.outby = request.user
+    else:
+        bag.outby = User.objects.filter(id=1).first()
```
[the remark this answers is not in the document]
Comment (Casassarnau) [reply baggage/views.py]: Web = true means it's not called from the API

[the remark this answers is not in the document]
Comment (casassg) [reply baggage/views.py]: why is outby the first user ever? This seems highly unsecure.

[the remark this answers is not in the document]
Comment (Casassarnau) [reply baggage/views.py]: It's beacasue we don't have an user API yet

[the remark this answers is not in the document]
Comment (Casassarnau) [reply baggage/views.py]: we can't know who makes the check-in

**Pull request merged**